tests/infrastructure/prometheus.go

```diff
@@ -180,7 +180,7 @@ var _ = DescribeInfra("[rfe_id:3187][crit:medium][vendor:cnv-qe@redhat.com][leve
 		var queryResult map[string]json.RawMessage
 
 		err = json.Unmarshal([]byte(stdout), &queryResult)
-		Expect(err).ToNot(HaveOccurred(), "failed to unmarshal query result")
+		Expect(err).ToNot(HaveOccurred(), "failed to unmarshal query result.", stdout)
```

Comment:
Nit:
```suggestion
		Expect(err).ToNot(HaveOccurred(), "failed to unmarshal query result: %s", stdout)
```
IIUC, your version will also print `stdout`, but as like "extraneous parameter, don't know what to do with it, here it is:".

Reply:
Done